src/cmake/thirdparty/SetupAxomThirdParty.cmake

```diff
@@ -180,13 +181,31 @@ endforeach()
 #------------------------------------------------------------------------------
 if (LUA_DIR)
     include(cmake/thirdparty/FindLUA.cmake)
-    blt_register_library(
+    blt_import_library(
         NAME          lua
         INCLUDES      ${LUA_INCLUDE_DIR}
         LIBRARIES     ${LUA_LIBRARY}
-        TREAT_INCLUDES_AS_SYSTEM ON)
+        TREAT_INCLUDES_AS_SYSTEM ON
+        EXPORTABLE    ON)
 else()
     message(STATUS "LUA support is OFF")
     set(LUA_FOUND OFF CACHE BOOL "")
 endif()
 
+
+#------------------------------------------------------------------------------
+# Targets that need to be exported but don't have a CMake config file
+#------------------------------------------------------------------------------
+set(TPL_DEPS cuda lua mfem mpi openmp)
```

Comment:
Would it be cleaner to build up `TPL_DEPS` as they are added as a `blt_import_library`?